### DISCUSSION TITLE
使用 Ant Design Vue 的 Modal.confirm 确认框时，出现了点击取消无法关闭的问题

### DISCUSSION THREAD

**Discussion by windMan778** [vbenjs/vue-vben-admin#3800] [General]
使用 [Ant](https://so.csdn.net/so/search?q=Ant&spm=1001.2101.3001.7020) Design Vue 的 Modal.confirm 确认框时，出现了点击取消和确定后 Modal.confirm 确认框无法关闭的问题

**Comment by windMan778** [comment]
版本应该没有问题 "ant-design-vue": "2.2.8","vue": "^3.2.20",

**Reply by MisterBoole** [reply]
ant-design-vue: 3.2.20  和 vue 3.2.X 搭配没有问题， 如果 vue 为  vue 3.4.X 就会有问题

**Comment by zqshier** [comment]
请问解决了吗

**Comment by rookie957** [comment]
请问解决了吗

**Comment by A-fairy** [comment]
为什么ant-design-vue: 3.2.17,  vue:  3.2.47, 这样按你们说版本也没冲突啊，也会有这个问题

**Reply by colonot** [reply]
"vue": "^3.3.13","ant-design-vue": "^3.2.0",可以
